README.md

```diff
@@ -4,3 +4,6 @@ Account linking service
 # Running Tests
 * `pip install nose nosegae nose-exclude`
 * ```nosetests --with-gae --gae-lib-root=`gcloud info --format="value(installation.sdk_root)"`/platform/google_appengine --exclude-dir=lib```
+
+# Running locally in Docker
+* `docker run -p=8080:8080 -p=8000:8000 quay.io/databiosphere/bond:latest`
```

Comment:
8000 is exposed and published...but for some reason it serves up an empty admin console

our APIs at 8080 do still work, though. so for now it's workable but i'd like to get this figured out